<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1031</width>
    <height>745</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_history">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>10</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>history</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_history">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>50</y>
      <width>301</width>
      <height>661</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>705</width>
      <height>577</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="pushButton_change">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/icons/change.png</normaloff>:/icons/change.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_left">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/icons/(.png</normaloff>:/icons/(.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_right">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/icons/).png</normaloff>:/icons/).png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_e">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/icons/e.png</normaloff>:/icons/e.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_pai">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/icons/pai.png</normaloff>:/icons/pai.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="pushButton_1">
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/icons/1.png</normaloff>:/icons/1.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="autoRepeatInterval">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_4">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/icons/4.png</normaloff>:/icons/4.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_7">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/icons/7.png</normaloff>:/icons/7.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_j">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/icons/j.png</normaloff>:/icons/j.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_angle">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/icons/jiao.png</normaloff>:/icons/jiao.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/icons/2.png</normaloff>:/icons/2.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_5">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/icons/5.png</normaloff>:/icons/5.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_8">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/icons/8.png</normaloff>:/icons/8.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_0">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/icons/0.png</normaloff>:/icons/0.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_ac">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/icons/ac.png</normaloff>:/icons/ac.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="pushButton_3">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/icons/3.png</normaloff>:/icons/3.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_6">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/icons/6.png</normaloff>:/icons/6.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_9">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/icons/9.png</normaloff>:/icons/9.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_dot">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/icons/..png</normaloff>:/icons/..png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_delete">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/icons/删除.png</normaloff>:/icons/删除.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="pushButton_add">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/icons/jia.png</normaloff>:/icons/jia.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_sub">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/icons/jian.png</normaloff>:/icons/jian.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_multi">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/icons/cheng.png</normaloff>:/icons/cheng.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_div">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/icons/chu.png</normaloff>:/icons/chu.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_output">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/icons/=.png</normaloff>:/icons/=.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_output">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>100</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Here's the answer.</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_input">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>681</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_about">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>10</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>about</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
